warcry_proj/warcry_proj/routes.py

```diff
@@ -0,0 +1,11 @@
+def includeme(config):
+    config.add_static_view('static', 'static', cache_max_age=3600)
+    config.add_route('home', '/')
+
+    # relevant routes for the Restful API
```

Comment:
Some general thoughts when dealing with REST api endpoints. You want to make sure you're as often as possible (if not always), using nouns as your endpoint names, and let the http verbs handle the "action".

So, for example, in a user based rest API, you would probably want:
To get all current users in the system:
GET /users/

To get information on a single user:
GET /users/<userid>

To create a new user:
POST /users/ (envelope with new user info)

To update an existing user:
PUT /users/<userid> (envelope with user info)

To update all users with a particular update:
PUT /users/ (envelope)

To delete a user:
DELETE /users/<userid>

To delete users with a particular set of criteria, or to delete all users:
DELETE /users/ (envelope with info)